controllers/funding/materials.js

```diff
@@ -394,58 +394,50 @@ function initForm({ router, routeConfig }) {
             const errors = validationResult(req);
 
             if (errors.isEmpty()) {
-                /**
-                 * Allow tests to run without sending email
-                 * this is only used in tests, so we confirm the form data was correct
-                 */
-                if (req.body.skipEmail) {
```

Comment:
Removing this check as a) the `mail` function already skips sending email when in test mode centrally and b) it feels better to test against the real user flow, even at the expense of asserting against the mail response.

Reply:
My only concern is that last year I introduced a bug where the material email sent out the names of the fields rather than their values (eg. `First name: yourName`) – I added the tests to at least verify the data being sent out was what we expected. Not sure if there's a way we could handle this here?